xhprof_lib/utils/Db/Mssql.php

```diff
@@ -1,46 +1,46 @@
 <?php
 
-/**
- * When setting the `id` column, consider the length of the prefix you're specifying in $this->prefix
- *
- *
- CREATE TABLE dbo.details
- (
- id nchar(17) NOT NULL,
- url nvarchar(255) NULL DEFAULT NULL,
- c_url nvarchar(255) NULL DEFAULT NULL,
- timestamp datetime NOT NULL DEFAULT getdate(),
- [server name] nvarchar(64) NULL DEFAULT NULL,
- perfdata nvarchar(max) NULL,
- type smallint NULL DEFAULT NULL,
- cookie nvarchar(max) NULL,
- post nvarchar(max) NULL,
- get nvarchar(max) NULL,
- pmu int NULL DEFAULT NULL,
- wt int NULL DEFAULT NULL,
- cpu int NULL DEFAULT NULL,
- server_id nchar(3) NOT NULL DEFAULT N't11',
- aggregateCalls_include nvarchar(255) NULL DEFAULT NULL,
- CONSTRAINT PK_details_id PRIMARY KEY (id)
- )
- GO
- CREATE NONCLUSTERED INDEX dbo.url
- ON dbo.details (url ASC)
- GO
- CREATE NONCLUSTERED INDEX dbo.c_url
- ON dbo.details (c_url ASC)
- GO
- CREATE NONCLUSTERED INDEX dbo.cpu
- ON dbo.details (cpu ASC)
- GO
- CREATE NONCLUSTERED INDEX dbo.wt
- ON dbo.details (wt ASC)
- GO
- CREATE NONCLUSTERED INDEX dbo.pmu
- ON dbo.details (pmu ASC)
- GO
- CREATE NONCLUSTERED INDEX dbo.timestamp
- ON dbo.details (timestamp
+/**
+ * When setting the `id` column, consider the length of the prefix you're specifying in $this->prefix
+ *
+ *
+ CREATE TABLE dbo.details
+ (
+ id nchar(17) NOT NULL,
+ url nvarchar(255) NULL DEFAULT NULL,
+ c_url nvarchar(255) NULL DEFAULT NULL,
+ timestamp datetime NOT NULL DEFAULT getdate(),
+ [server name] nvarchar(64) NULL DEFAULT NULL,
+ perfdata nvarchar(max) NULL,
+ type smallint NULL DEFAULT NULL,
+ cookie nvarchar(max) NULL,
+ post nvarchar(max) NULL,
+ get nvarchar(max) NULL,
+ pmu int NULL DEFAULT NULL,
+ wt int NULL DEFAULT NULL,
+ cpu int NULL DEFAULT NULL,
+ server_id nchar(17) NOT NULL DEFAULT N't11',
```

Comment:
The problem comes from mass-replacement of line endings from `\r\n` into `\n` it seems.